pkgs/by-name/dw/dwlb/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, writeText
+, pkg-config
+, wayland
+, wayland-protocols
+, pixman
+, fcft
+, conf ? null
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dwlb";
+  version = "unstable-2024-05-17";
+
+  src = fetchFromGitHub {
+    owner = "kolunmi";
+    repo = "dwlb";
+    rev = "master";
```

```suggestion
    rev = "0daa1c1fdd82c4d790e477bf171e23ca2fdfa0cb";
```

You need to "pin" the package to a certain commit if it's going to be an unstable version